src/ingestion/models.py

```
@@ -0,0 +1,13 @@
+from django.db import models
+
+class TicketEvent(models.Model):
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)